config.json

```diff
@@ -719,9 +593,13 @@
       "uuid": "dec66f89-39d0-4857-9679-a035cf4259d7",
       "slug": "octal",
       "deprecated": true
+    },
+    {
+      "uuid": "0e43944b-0a68-5680-ef11-70999d2df897c894476",
+      "slug": "twelve-days",
+      "core": false,
+      "difficulty": 4
```

Comment:
OK, @AakashMallik I only just noticed that there are no "topics" for your exercise.

None of the other tracks I checked had any either 😮.  However, there is a similar exercise (`beer-song`) that @rchavarria tagged with these topics on the JavaScript track-- I think these could work:

```json
      "topics": [
        "Control flow (conditionals)",
        "Control flow (loops)",
        "Strings"
```

Would you mind adding those, or if you think other topics would be more appropriate let me know.